powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/internal/LambdaLoggingAspect.java

```diff
@@ -305,49 +303,45 @@ private double samplingRate(final Logging logging) {
         return logging.samplingRate();
     }
 
+    @SuppressWarnings("java:S3457")
     private void logRequestHandlerEvent(final ProceedingJoinPoint pjp, final Object event) {
         Logger log = logger(pjp);
         if (log.isInfoEnabled()) {
-            LoggingUtils.logMessagesAsJson(true);
-            asJson(event).ifPresent(log::info);
-            LoggingUtils.logMessagesAsJson(false);
+            log.info("Handler Event", entry("event", event));
         }
     }
 
+    @SuppressWarnings("java:S3457")
     private Object[] logRequestStreamHandlerEvent(final ProceedingJoinPoint pjp) {
         Object[] args = pjp.getArgs();
         Logger log = logger(pjp);
         if (log.isInfoEnabled()) {
-            LoggingUtils.logMessagesAsJson(true);
             try {
                 byte[] bytes = bytesFromInputStreamSafely((InputStream) pjp.getArgs()[0]);
                 args[0] = new ByteArrayInputStream(bytes);
                 // do not log asJson as it can be something else (String, XML...)
-                log.info("{}", new String(bytes, UTF_8));
+                log.info("Handler Event", entry("event", new String(bytes, UTF_8)));
             } catch (IOException e) {
                 LOG.warn("Failed to log event from supplied input stream.", e);
             }
-            LoggingUtils.logMessagesAsJson(false);
         }
         return args;
     }
 
+    @SuppressWarnings("java:S3457")
     private void logRequestHandlerResponse(final ProceedingJoinPoint pjp, final Object response) {
         Logger log = logger(pjp);
         if (log.isInfoEnabled()) {
-            LoggingUtils.logMessagesAsJson(true);
-            asJson(response).ifPresent(log::info);
-            LoggingUtils.logMessagesAsJson(false);
+            log.info("Handler Response", entry("response", response));
```

Comment:
for information, it will log:
```json
{
  "message": "Handler Response",
  "response": {
    // ...
  }
}
```
should we name it differently? should we log another message instead?